requirements.txt

```diff
@@ -0,0 +1,3 @@
+aiohttp
+cloudflare
```

Comment:
Need PR 76 Support API token , perhaps you can you add >= 2.3.1

Reply:
why? it was working properly on my local test

Reply:
Yes, It's works now, because cloudflare just release 2.3.1. a few days ago.
but you can just add
clouflare >= 2.3.1

Reply:
see https://github.com/criteo/cloudflare-exporter/pull/3